BonovoBluetooth/src/com/bonovo/bluetooth/BonovoBluetoothHandfree.java

```diff
@@ -359,49 +387,59 @@ protected void onResume() {
 			}
 		}
 		mDialpadPressCount = 0;
+		

```

Comment:
Sorry, i'll try to remember not to do that again.
Had a problem tonight with the Chinese text in one of the CPP files getting garbled.  Sorry for the noise but I'm pretty sure everything is back in order now.